drivers/storage/scsiport/scsiport.c

```diff
@@ -2815,6 +2815,11 @@ ScsiPortDeviceControl(IN PDEVICE_OBJECT DeviceObject,
 
     switch (Stack->Parameters.DeviceIoControl.IoControlCode)
     {
+      case IOCTL_ATA_PASS_THROUGH:
+          DPRINT1("IOCTL_ATA_PASS_THROUGH unimplemented. (Caller should use IOCTL_SCSI_PASS_THROUGH instead.)\n");
+          Status = STATUS_NOT_IMPLEMENTED;
+          break;
```

Comment:
Please rebase on top of master, and move the case all the way down to `default` one.

Reply:
Done.